src/eko/evolution_operator/grid.py

```diff
@@ -153,7 +154,7 @@ def get_threshold_operators(self, path):
         The internal dictionary is self._threshold_operators and its structure is:
         (q2_from, q2_to) -> eko.operators.Operator
 
-        It computes and stores the necessary macthing operators
+        It Compute and stores the necessary macthing operators
```

Comment:
```suggestion
        It computes and stores the necessary macthing operators
```
the search-and-replace pattern failed :upside_down_face: